steps/create-a-new-release-in-jira/1.0.0/step.yml

```diff
@@ -0,0 +1,100 @@
+title: Create a new release in JIRA
+summary: |
+  Create a new release in JIRA.
+description: |
+  This steps create a Release in your JIRA project, with the start date, publication and description you need.
+website: https://github.com/Emalober/bitrise-step-create-a-new-release-in-jira
+source_code_url: https://github.com/Emalober/bitrise-step-create-a-new-release-in-jira
+support_url: https://github.com/Emalober/bitrise-step-create-a-new-release-in-jira/issues
+published_at: 2021-10-06T21:03:16.937402-03:00
+source:
+  git: https://github.com/Emalober/bitrise-step-create-a-new-release-in-jira.git
+  commit: 567ef82a59926847431c10b3256274ac2ad35bb9
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    description: |
+      Jira project prefix to search for tasks.
+
+      e.g. "PROJ"
+    is_expand: true
+    is_required: true
+    summary: Jira project prefix to search for tasks.
+    title: Jira project prefix
+  project_prefix: null
+- jira_domain: null
+  opts:
+    description: |
+      Jira company domain.
+
+      e.g. "example.atlassian.net"
+    is_expand: true
+    is_required: true
+    summary: Jira company domain
+    title: Jira domain
+- jira_user: null
+  opts:
+    description: |
+      Jira user used to authenticate with Jira API
+
+      e.g. "someone@example.pl"
+    is_sensitive: true
```

Comment:
I don't think that username should be sensitive.
What is a reason behind that?

Reply:
I saw that in many cases, it is used as a sensitive field.